bin/combinetf2_plot_hists_uncertainties.py

```diff
@@ -421,17 +428,19 @@ def make_plots(
 
     hist_total = result["hist_postfit_inclusive"].get()
 
+    if hist_impacts is not None:
+        uncertainties = np.array(hist_impacts.axes["impacts"], dtype=str)
```

Comment:
I realize the code was like this before, but why does this actually need to be an np.array?

Reply:
Yes this is to get an array of the bin labels for a StrCategory axis. The hist_impacts.axes["impacts"] gives a StrCategory object which doesn't support things like slicing, e.g. "[:]" that is used later.

Reply:
right but regular python strings should support slicing operations as well

Reply:
You mean list? It should be a list instead of a np.array?

Reply:
Ah yes right, it should be a list in this case

```list(hist_impacts.axes["impacts"])``` should work

Reply:
I'll merge as-is in any case and this can be adjusted later.